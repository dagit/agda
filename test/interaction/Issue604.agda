module Issue604 where

f : (y : Set1) → (x : Set) → Set1
f x = ?
