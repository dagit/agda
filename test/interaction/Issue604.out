(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue604 (Issue604.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue604.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set → Set₁\n" nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-give-action 0 "λ x₁ → ?")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?1 : Set₁\n" nil)
((last . 1) . (agda2-goals-action '(1)))
(agda2-give-action 1 "x")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
